spec/controllers/admin/attachments_controller_spec.rb

```diff
@@ -47,7 +47,7 @@ module Alchemy
 
         context 'with params[:only]' do
           it 'only loads attachments with matching content type' do
-            get :index, only: 'jpeg'
+            alchemy_get :index, only: 'jpeg'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.